### ISSUE TITLE
NTR: N-carbamyl-beta-aminoisobutyric acid high in CSF (Lab: cerebrospinal fluid tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8467]
New term request N-carbamyl-beta-aminoisobutyric acid high in CSF (elthat_111121052102): Cerebrospinal fluid

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000209
- label: Elevated CSF N-carbamyl-beta-aminoisobutyric acid concentration
- parents: [Abnormal CSF metabolite concentration (HP:0025454)](https://hpo.jax.org/app/browse/term/HP:0025454)
- pmid: PMID:15385443
- def: The concentration of N-carbamyl-beta-aminoisobutyric acid in the cerebrospinal fluid (CSF) is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8467
- synonyms: N-carbamyl-beta-aminoisobutyric acid high in CSF(NONE)
